src/main/java/algorithms/equivalencematching/LexicalEquivalenceMatcherSigmoid.java

```diff
@@ -41,6 +41,21 @@
 	int slope;
 	double rangeMin;
 	double rangeMax;
+
+	public static void main(String[] args) throws OWLOntologyCreationException, AlignmentException {
+		File ontoFile1 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/bibframe.rdf");
+		//File ontoFile2 = new File("./files/_PHD_EVALUATION/BIBFRAME-SCHEMAORG/ONTOLOGIES/schema-org.owl");
+		//File ontoFile1 = new File("./files/_PHD_EVALUATION/OAEI2011/ONTOLOGIES/301302/301302-301.rdf");
+		File ontoFile2 = new File("./files/_PHD_EVALUATION/OAEI2011/ONTOLOGIES/301302/301302-302.rdf");
+		final int slope = 3;
+		final double rangeMin = 0.5;
+		final double rangeMax = 0.7;
```

Comment:
Why not just get these from AlgorithmSettings?

Reply:
Fixed it in the latest commit